tests/unit/test_charm.py

```diff
@@ -636,15 +425,15 @@ def test_theme_reconciliation(self):
         self.harness.update_config({"themes": "123, abc"})
 
         self.assertEqual(
-            self.patch.installed_themes(),
+            self.patch.container.installed_themes,
             set(self.harness.charm._WORDPRESS_DEFAULT_THEMES + ["abc", "123"]),
             "adding themes to themes config should install trigger theme installation",
```

Comment:
Minor detail: should trigger theme installation?

Reply:
Fixed, thanks!